js/mainjs/trayMenu.js

```diff
@@ -8,15 +8,22 @@ module.exports = function (window) {
 	var menutemplate = [
 		{
 			label: 'Show Sia',
-			click: function () { window.show(); }
+			click: function () { window.restore(); }
 		},
 		{ type: 'separator' },
 		{
 			label: 'Hide Sia',
-			click: function () { window.hide(); }
+			click: function () { window.minimize(); }
 		},
+		{ type: 'separator' },
+		{
+			label: 'Quit Sia',
+			click: function() {
+				window.wantsQuit = true;
+				window.close();
```

Comment:
According to the docs, destroy does this:

`Force closing the window, the unload and beforeunload event won't be emitted for the web page, and close event will also not be emitted for this window, but it guarantees the closed event will be emitted.`

This means any event listeners assigned to unload, beforeunload, and close() wont fire.  There's two listeners for these events in Sia-UI,
`window.onbeforeunload = closeLog` .
in uiManager.js at L155 and
`mainWindow.on('close'...` 
in initWindow.js at L36

Neither of these get called if you just call `.destroy()`.